users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class User(models.Model):
+    email        = models.CharField(max_length=300)
+    phone_number = models.CharField(max_length=300)
+    name         = models.CharField(max_length=300)
+    birthday     = models.CharField(max_length=300)    
```

Comment:
`DateField`를 사용해야되지 않을까요?
```suggestion
    birthday     = models.CharField(max_length=300)    
```

Reply:
카카오로 정보를 받아와 CharField로 정보를 받는다고 생각했습니다. 수정하겠습니다~